mmv1/third_party/terraform/fwmodels/provider_model.go.erb

```diff
@@ -53,7 +53,6 @@ type ProviderModel struct {
 	CloudResourceManagerCustomEndpoint types.String `tfsdk:"cloud_resource_manager_custom_endpoint"`
 	EventarcCustomEndpoint             types.String `tfsdk:"eventarc_custom_endpoint"`
 	FirebaserulesCustomEndpoint        types.String `tfsdk:"firebaserules_custom_endpoint"`
-	NetworkConnectivityCustomEndpoint  types.String `tfsdk:"network_connectivity_custom_endpoint"`
```

Comment:
Reminder for myself to follow up on the DCL piece you mentioned. From looking at the generated diffs, I think what you've done is fine, but want to make sure.

Reply:
I think this is technically ok, but it would be safer to add a `tpgtools_product.yaml` file to `tpgtools/overrides/networkconnectivity` that looks like [this](https://github.com/GoogleCloudPlatform/magic-modules/pull/6608/files#diff-de1e23c69df2f622c191e8c6a721f08bbcbdda0cdb3bd1d1b4c4343e9e648816). Could you try that out, in addition to this line? I think it will clean up some other code that is now redundant.

Reply:
@roaks3 , great, added the file that you mention.